plugins/calibre.koplugin/wireless.lua

```diff
@@ -644,9 +644,20 @@ end
 
 function CalibreWireless:sendToCalibre(arg)
     logger.dbg("GET_BOOK_FILE_SEGMENT", arg)
-    -- not implemented yet, we just send an invalid opcode to raise a control error in calibre.
-    -- If we don't do this calibre will wait *a lot* for the file(s)
-    self:sendJsonData('NOOP', {})
+    local inbox_dir = G_reader_settings:readSetting("inbox_dir")
+    local path = inbox_dir .. "/" .. arg.lpath
+
+    local file = io.open(path, "rb")
+    if file == nil then
```

Comment:
or if not file

Reply:
:+1: for `if not file`. Sounds more idiomatic.